assets/js/validation.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+
+let nameInput = document.querySelector('#name');
+let projectInput = document.querySelector('#project');
+let nameSpan = document.querySelector('#name-result');
+let projectSpan = document.querySelector('#project-result');
+
+
+const validateName = function(){
+    nameSpan.innerText='...';
+    axios.post(nameSpan.dataset.path, {input: nameInput.value})
+        .then(response => {
+            if (response.data.valid) {
+                nameSpan.innerHTML = ":)";
+            } else {
+                nameSpan.innerHTML = ":(";
+            }
+        })
+        .catch(error => console.log(error))
```

Comment:
Verta klaidos pranešimą atspausdinti į žmogui matomą vietą (t.y. HTML elementą). Nes testuojant puslapį **su telefonu** yra labai sunku pasileisti jo `console` ir patikrinti rezultatą.

Jei jau naudoji `console` – verta išskirti klaidas ir išsaugoti klaidos kelią (angl. _trace_):
```suggestion
        .catch(error => console.error(error))
```
![image](https://user-images.githubusercontent.com/1453957/57180041-3202d000-6e8d-11e9-9619-2977b466fa09.png)
:vs: 
![image](https://user-images.githubusercontent.com/1453957/57180051-5d85ba80-6e8d-11e9-969e-8c93cfde42bf.png)

P.S. Buvo laikai, kai kažkurios naršyklės neturėjo `console` objekto ir dėl to lūždavo puslapiai. Šiais lakais poveikis labai retai kada matomas.